shared-data/python/opentrons_shared_data/liquid_classes/liquid_class_definition.py

```diff
@@ -0,0 +1,347 @@
+        ..., description="Submerge settings for single dispense."
+    )
+    retract: RetractDispense = Field(
+        ..., description="Pipette retract settings after a single dispense."
+    )
+    positionReference: PositionReference = Field(
+        ..., description="Position reference for single dispense."
+    )
+    offset: Coordinate = Field(..., description="Relative offset for single dispense.")
+    flowRateByVolume: LiquidHandlingPropertyByVolume = Field(
+        ...,
+        description="Settings for flow rate keyed by target aspiration/dispense volume.",
+    )
+    mix: MixProperties = Field(..., description="Mixing settings for after an aspirate")
+    pushOutByVolume: LiquidHandlingPropertyByVolume = Field(
+        ..., description="Settings for pushout keyed by target aspiration volume."
+    )
+    delay: DelayProperties = Field(..., description="Delay after dispense, in seconds.")
+
+
+class MultiDispenseParams(BaseModel):
+    """Parameters specific to the multi-dispense function."""
+
+    submerge: Submerge = Field(..., description="Submerge settings for multi-dispense.")
+    retract: RetractDispense = Field(
+        ..., description="Pipette retract settings after a multi-dispense."
+    )
+    positionReference: PositionReference = Field(
+        ..., description="Position reference for multi-dispense."
+    )
+    offset: Coordinate = Field(
+        ..., description="Relative offset for single multi-dispense."
+    )
+    flowRateByVolume: LiquidHandlingPropertyByVolume = Field(
+        ...,
+        description="Settings for flow rate keyed by target aspiration/dispense volume.",
+    )
+    mix: MixProperties = Field(..., description="Mixing settings for after an aspirate")
+    conditioningByVolume: LiquidHandlingPropertyByVolume = Field(
+        ...,
+        description="Settings for conditioning volume keyed by target dispense volume.",
+    )
+    disposalByVolume: LiquidHandlingPropertyByVolume = Field(
+        ..., description="Settings for disposal volume keyed by target dispense volume."
+    )
+    delay: DelayProperties = Field(..., description="Delay settings after an aspirate")
+
+
+class ByTipTypeSetting(BaseModel):
+    """Settings for each kind of tip this pipette can use."""
+
+    tipType: str = Field(
+        ...,
+        description="The tip type whose properties will be used when handling this specific liquid class with this pipette",
+    )
+    aspirate: AspirateParams = Field(
+        ..., description="Aspirate parameters for this tip type."
+    )
+    singleDispense: SingleDispenseParams = Field(
+        ..., description="Single dispense parameters for this tip type."
+    )
+    multiDispense: Optional[MultiDispenseParams] = Field(
+        None, description="Optional multi-dispense parameters for this tip type."
+    )
+
+
+class ByPipetteSetting(BaseModel):
+    """The settings for a specific kind of pipette when interacting with this liquid class."""
```

Comment:
Since this is part of a liquid class definition (as opposed to pipette definition), how about -
```suggestion
    """The settings for this liquid class when used with a specific kind of pipette."""
```
This matches with your description for `byPipette` on line 346